apps/frontend/src/libs/components/input/input.tsx

```diff
@@ -44,10 +46,22 @@ const Input = <T extends FieldValues>({
 	const hasLeftIcon = Boolean(leftIcon);
 	const hasRightIcon = Boolean(rightIcon);
 	const isTextArea = Boolean(rowsCount);
+	const inputClassNames = getValidClassNames(
+		styles["input-field"],
+		isTextArea && styles["input-textarea"],
+		hasLeftIcon && styles["with-left-icon"],
+		hasRightIcon && styles["with-right-icon"],
+	);
 
 	return (
 		<label className={styles["input-label"]}>
-			<span className={styles["input-label-text"]}>{label}</span>
+			<span
+				className={
+					isLabelHidden ? "visually-hidden" : styles["input-label-text"]
+				}
```

Comment:
"input-label-text" is a default style which should be always present
```suggestion
				className={
				  getValidClassNames(
				    styles["input-label-text"],
				    isLabelHidden && "visually-hidden")
				}
```

Reply:
Oh, yeah, changed